pkg/data/components/EmaThroughput.yaml

```diff
@@ -55,8 +55,8 @@ properties:
       traffic patterns, but the more volatile the sampling rate will be.
     type: int
     validations:
-      - positive
-    default: 60
+      - duration
```

Comment:
This isn't currently valid, because we don't yet have a duration editor in the UI, and type "int" will prevent entry of the duration suffix.

We can change the type to string, but I would prefer that we leave it as an int and append the "s" in the template.

Also, I'm in the middle of a redesign of all the refinery stuff, so please don't stress about this component too much. It's almost certainly changing rather radically.

Reply:
Yeah, makes sense to use int and append the s as part writing the config. I'll update it. Thanks.

Reply:
I've reverted back to using the positive validation. I got it to work with appending the `s` as a suffix, but had to remove the `encodeAsInt` operator otherwise it would fail to write the value correctly.

Example output:
```
Diff:
    --- Expected
    +++ Actual
    @@ -4,3 +4,3 @@
    EMAThroughputSampler:
    -            AdjustmentInterval: 60s
    +            AdjustmentInterval: "int\x1D60s"
```